tests/test_lib_caconnector.py

```diff
@@ -280,6 +281,25 @@ def test_05_templates(self):
         self.assertTrue(ddiff.days > 740, ddiff.days)
         self.assertTrue(ddiff.days < 760, ddiff.days)
 
+    def test_99_cleanup(self):
+        filelist = glob.glob("{0!s}/100*.pem".format(WORKINGDIR))
+        for f in filelist:
+            os.remove(f)
+
+        FILES = ["cornelius_user@localhost.localdomain_realm1.pem",
+                 "cornelius_user@localhost.localdomain_realm1.req",
+                 "DE_Hessen_privacyidea_requester.localdomain.req",
```

Comment:
This one is currently tracked and should be removed from git.
Also it's corresponding PEM-File (`DE_Hessen_privacyidea_requester.localdomain.pem`) should be removed here.

Reply:
will do.

Reply:
done in 151a9d2d